src/containers/shared/components/Transaction/SetHook/test/SetHookSimple.test.tsx

```diff
@@ -0,0 +1,81 @@
+import { createSimpleWrapperFactory } from '../../test/createWrapperFactory'
+import { Simple } from '../Simple'
+import mockSetHook from './mock_data/SetHook.json'
+import mockSetHook2 from './mock_data/SetHook2.json'
+import { expectSimpleRowText } from '../../test/expectations'
+
+const createWrapper = createSimpleWrapperFactory(Simple)
+
+describe('SetHookSimple', () => {
```

Comment:
Please add a test with a failing transaction especially with the metadata being parsed (a common issue).